scripts/i18next-parser.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  contextSeparator: '_',
+  createOldCatalogs: false,
+  keySeparator: false,
+  locales: ['en', 'ja'],
+  defaultNS: 'assisted_installer', // the default file for strings when using useTranslation, etc
```

Comment:
BTW, you could do the same thing you do in the hook here:
```suggestion
  defaultNS: process.env.TRANSLATION_NAMESPACE ?? 'assisted_installer', // the default file for strings when using useTranslation, etc
```

Reply:
ok!